src/nvim/ops.c

```diff
@@ -5372,11 +5372,10 @@ void cursor_pos_info(dict_T *dict)
       }
     }
 
-    // Don't shorten this message, the user asked for it.
     bom_count = bomb_size();
     if (bom_count > 0) {
-      vim_snprintf((char *)IObuff + STRLEN(IObuff), IOSIZE - STRLEN(IObuff),
```

Comment:
`IOSIZE - STRLEN(IObuff)` avoids buffer overrun. It should stay. 

Perhaps @jbradaric can review these changes since he gave this attention recently.